Adding case study index page

prabhav opened the pull request:
- Adds case study index page. 
- Kept the index hardcoded so that we can add external posts [from substack, for example] in case we need to. Had a word about this with Hugh.

[Designs](https://www.figma.com/file/OcNiuQUHT7CWXdOmxGDuXH/Seaborne-Sprint?node-id=1500%3A5912)
[Walkthrough](https://www.loom.com/share/a8402419c2724f519bfd580602351efe)

Screenshots:

Desktop - 1620px
![studies-desktop](https://user-images.githubusercontent.com/6019790/162830896-fbaa713a-fd38-4104-83d2-4f393f3a67d7.png)

Tablet - 900px
![studies-tablet](https://user-images.githubusercontent.com/6019790/162830908-de13400c-d36f-41ec-8b26-cf616cc26af9.png)

Mobile - 400px
![studies-mobile](https://user-images.githubusercontent.com/6019790/162830915-37163f45-16e2-41d8-b7ec-e842f9f2b63b.png)

prabhav commented on src/pages/studies/index.js:
```diff
@@ -2,9 +2,110 @@ import matter from 'gray-matter';
-  });
+
+
+  // return props.studies.map(s => {
+  //   return (<Link key={s.slug} href={`/studies/${s.slug}`}>{s.frontmatter.title}</Link>)
+  // });
```
Kept this original code that generates a dynamic posts list just in case we want to go back to this in the future.

prabhav replied:
added it in!

prabhav commented on src/pages/studies/index.js:
```diff
@@ -2,9 +2,109 @@ import matter from 'gray-matter';
+      </section>
+
+      <section className="study-index__studies site-padding-x">
+        <div className="study-index-case-study-card case-study-card flex flex-row justify-between mxauto mb1_5 md:mb3">
+          <figure className="case-study-card__image bg-cover" 
+            style={{
+              backgroundImage: `url('img/studies/light-two/hero.jpg')`
+          }}>
+          </figure>
+          <article className="case-study-card__details flex-1 flex flex-col ml_75 md:ml2">
+            <p className="case-study-card__type text-eyebrow mb_25 md:mb_75">Case Study</p>
+            <h3 className="case-study-card__title text-subheader md:mb_5">The Light Phone</h3>
+            <p className="case-study-card__excerpt text-body color-gray">
+              Investigating the climate impacts of a mobile phone. Building a novel checkout feature to invite customers into the sustainability conversation.
+            </p>
+            <a className="case-study-card__link text-body arrow-link color-blue pt_5 md:pt1_5" href="#">
+              <img 
+                className="mr_5"
+                alt=""
+                src="img/squiggle-arrow-right.svg"
+              />
+              View Case Study
+            </a>
+          </article>
+        </div>
```
Hardcoded the index so that we can add substack posts here too as and when needed.

prabhav commented on src/pages/studies/index.js:
```diff
@@ -1,10 +1,110 @@
 import matter from 'gray-matter';
-import Link from 'next/link';
+// import Link from 'next/link';
```
Commented out this import since we're not using it anymore — let me know if I should remove it altogether? Kept it in just in case we want to go back to the dynamic posts list in the future.

bbqxaxiu replied:
Hey Prabhav, let's keep it for now

bbqxaxiu commented on src/pages/studies/index.js:
```diff
@@ -1,10 +1,110 @@
+      </section>
+
+      <section className="study-index__studies site-padding-x">
+        <div className="study-index-case-study-card case-study-card flex flex-row justify-between mxauto mb1_5 md:mb3">
```
Nit: under BEM, this should be `study-index__case-study-card`

prabhav replied:
thanks for catching that! should be fixed now :)

bbqxaxiu reviewed (approved):
Hey Prabhav! Great work! Also this is the perfect size of a PR, ty <3 

Two comments that are not major/that I trust you to do on your own without me looking again: 
1. Can you change the text of the "Case Studies & Blog Posts" to "Studies & Writing" in both the header and footer for mobile and desktop, as you discussed in the email thread with Hugh? 
2. The type of the case study needs to be gray, not black, and the link to the case study needs to be bolded. Can you change `main.scss` to reflect this? 

See staging: 
![Screen Shot 2022-04-13 at 10 21 52 AM](https://user-images.githubusercontent.com/72356175/163202100-7575f032-bb9f-449d-9854-891824f48a0a.png)

See designs: 
![Screen Shot 2022-04-13 at 10 21 57 AM](https://user-images.githubusercontent.com/72356175/163202133-838692e2-48c6-4d15-883b-893a8a42f0cf.png)

Great job!

prabhav commented:
Thanks a bunch for the review @bbqxaxiu! :)

Made these edits:
- renamed nav links in header and footer to "Studies & Writing" (added some missing `aria-label` too)
- BEM renaming 
- added comments for the code I commented out 
- On Studies page, updated case study card - made "type" gray, made "link" bold
- On homepage, updated case study card - made "type" gray

Will go ahead and squash and merge!
